README.md

```diff
@@ -270,7 +311,7 @@ db_password:
 ```
 
 ### 1Password Lookup
-An Alternative to the alternative secret store is accessing 1password secrets using the 1password cli (`op`).
+An alternative to the alternative secret store is accessing 1password secrets using the 1password cli (`op`).
```

Comment:
This sentence could use some work 😂

Reply:
Not sure how that change got in there :/

Fixed in 4ad4ade